### ISSUE TITLE
Blue buttons lack halo on keyboard focus

### ISSUE THREAD

**Issue by ryanfeeley** [mozilla/fxa-content-server#5179]
STEPS TO REPRODUCE
1. In Settings tab through sections with your keyboard

EXPECTED RESULTS
* All section buttons will have focus halo

ACTUAL RESULTS
* Blue buttons don't have focus halo

![focus](https://user-images.githubusercontent.com/68213/27708431-c67603c0-5ccd-11e7-9f3a-82dd3f48e5ec.gif)
